src/components/profile/viewProfile.jsx

```diff
@@ -221,6 +239,57 @@ class ViewProfile extends Component {
                     then type all required information
                   </div>
                 )}
+                <div className='bookmark'>
+                  <h1>Bookmarks</h1>
+                  {isBookmarksFetched === 'done' && Bookmarks.length > 0 ? (
+                    Bookmarks.map(bookmark => {
```

Comment:
Where is `Bookmarks` coming from?

Reply:
```Bookmarks``` is coming from props. if you can check on line 143 of ```viewProfile.jsx```  where the render method starts, I destructure ```bookmarks``` from ```this.props``` then on line 145 I destructure ```Bookmarks``` from ```bookmarks```.